qa/tasks/cephfs/test_mds_metrics.py

```diff
@@ -497,15 +496,15 @@ def test_perf_stats_stale_metrics(self):
             self._cleanup_test_dirs()
 
     def test_client_metrics_and_metadata(self):
-        self.mount_a.umount_wait()
-        self.mount_b.umount_wait()
-        self.fs.delete_all_filesystems()
+        self.mount_a.umount_wait(force=True)
+        self.mount_b.umount_wait(force=True)
+        self.mds_cluster.delete_all_filesystems()
 
         self.run_ceph_cmd("fs", "flag", "set", "enable_multiple",
             "true", "--yes-i-really-mean-it")
 
         # creating filesystem
-        fs_a = self._setup_fs(fs_name="fs1")
+        fs_a = self._setup_fs(fs_name="fs1", client_id=self.mount_a.client_id)
```

Comment:
Sorry, but how is this fixing the issuee described in https://tracker.ceph.com/issues/68446 ?

Once client auth caps is configured and mounted, reconfiguring the auth caps for the client will no effect the mount, isn't it?

Reply:
@joscollin This commit was marked resolved without an explanation. Unresolving.